Userland/Utilities/mv.cpp

```diff
@@ -64,6 +66,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             new_path = combined_new_path.characters();
         }
 
+        if (no_clobber && stat(new_path.characters(), &st) == 0)
```

Comment:
Is it dangerous that stat might return other error?
In which case we might still continue even though the file exists?

Reply:
I'm not familiar enough with stat to have an informed opinion about the risk. Happy to change this if there's a more reliable way to check whether a file exists.

Reply:
I don't know, I guess you can use `Core::File::exists(new_path)`.
It currently does the same but if we run into problems we can then fix it everywhere that method is used

Reply:
Sounds good. I've made this change.